.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Check
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check_duplicate_runs:
+    name: Check for duplicate runs
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          concurrent_skipping: always
+          cancel_others: true
+          skip_after_successful_duplicate: true
+          paths_ignore: '["**/README.md", "**/CHANGELOG.md", "**/LICENSE"]'
+          do_not_skip: '["pull_request"]'
+
+  tests:
+    name: Run tests (Elixir ${{matrix.elixir}}, OTP ${{matrix.otp}})
+    needs: check_duplicate_runs
+    if: ${{ needs.check_duplicate_runs.outputs.should_skip != 'true' }}
+
+    strategy:
+      matrix:
+        elixir:
+        - "1.7"
+        - "1.8"
+        - "1.9"
+        - "1.10"
+        - "1.11"
+        otp:
```

Comment:
There is no OTP 19 because it ubuntu-18.04 and ubuntu-20.04 were updated fex months ago and become incompatible with an installer. Also phoenix repo CI/CD is running tests only for 21-23 OTP versions, so there is actually no need to test depending repo against older OTP versions.